FrameworkSystem/scripts/dirac-populate-component-db.py

```diff
@@ -0,0 +1,147 @@
+#!/usr/bin/env python
+
+"""
+Populates the database with the current installations of components
+Requires the local dirac.cfg to contain the full list of hosts
+under Registry/Hosts
+"""
```

Comment:
Well, I believe this is not fully true: the list will be taken from the central configuration service, at least if this script is run from a standard client installation.